controllers/proverbs-controllers.js

```diff
@@ -0,0 +1,73 @@
+const { validationResult } = require('express-validator');
+
+const Proverb = require('../models/proverb');
+const user = require('../models/user');
+
+const getProverbs = async (req, res, next) => {
+	res.send('hi from prov bro ;) ');
+};
+
+const postProverb = async (req, res, next) => {
+	const errors = validationResult(req);
+	if (!errors.isEmpty()) {
+		return next(errors);
+	}
+
+	const { proverb, literalTranslation: translation, explanation, userId } = req.body;
+
+	const postedProverb = new Proverb({
+		proverb,
+		translation,
+		explanation,
+		userId
+	});
+
+	if (userId) {
+		let user;
+		try {
```

Comment:
I would move this logic to another function called getUserById or something like that so you could reuse it in more places and the postProverb method would be focused on creating proverbs.